services/minespace-web/common/actionCreators/projectActionCreator.ts

```diff
@@ -505,6 +507,55 @@ export const removeDocumentFromMajorMineApplication = (
     .finally(() => dispatch(hideLoading()));
 };
 
+export const documentsCompression = (
+  mineGuid: string,
```

Comment:
This looks really generic! I'm wondering, is this a function that other documents can use or is it specific to projects?

Reply:
Ah I put it in projects because from what I know so far, the compression only happens in projects. Is there a better location you would suggest for this?

Reply:
Currently it only happens in projects but since there's nothing that makes it so that _only_ project documents can be compressed, I would put it in documentActionCreator. If we don't have it already I would make it.

Reply:
The documentCompression and the pollDocument function have been moved to documentActionCreator now.